woocommerce/google-listings-and-ads
# Bump TravisCI's OS and node version to match the one used for the release.

### Changes proposed in this Pull Request:

Bumps:
- OS to ~`Focal Ubuntu 20.04.2 LTS`~ `Bionic Ubuntu Ubuntu 18.04.5 LTS`
- node to `12.21.0`
To ~match~ _get closer to_ the versions used by @mikkamp for the release


### Screenshots:


### Detailed test instructions:

1. Go to [Travis](https://travis-ci.com/github/woocommerce/google-listings-and-ads/builds/227238431)
2. Check it builds with success
3. Checkout and build locally
4. Make sure it works with the version set by `.nvmrc`

:warning: With a higher version of node: `12.22.1` some `node-sass` problems were reported
:warning: We use `Focal Ubuntu 20.04.2 LTS` locally, but it does not support PHP 7.3 on Travis
